### ISSUE TITLE
Starter app doesn't work on Windows.

### ISSUE THREAD

**Issue by sherif-elmetainy** [BuilderIO/qwik#280]
I am using 
Windows 10 version 21H2
Node version 16.14.0
npm version 8.3.1

I ran the following command
```bash 
npm init qwik@latest
```
Then I chose default options (pressed enter)
Then
```bash
cd qwik-app
npm install
npm start
```
I didn't get an error during that process.
I opened http://localhost:3000 in Google Chrome Version 99.0.4844.51 (Official Build) (64-bit)
The I got a blank webpage
I found the following errors in the chrome's developer console:

```
GET http://localhost:3000/src/h_main_main_onmount.js net::ERR_ABORTED 404 (Not Found)
    (anonymous) @ main.tsx:4
    qrlImport @ qrl.public.js:21
    (anonymous) @ component.public.js:305
    Promise.then (async)
    resolveQrl @ component.public.js:304
    onRenderFactory @ component.public.js:229
    qPropWriteQRL @ props-on.js:85
    setEvent @ props.js:32
    updateProperties @ cursor.js:484
    createElm @ cursor.js:345
    updateChildren @ cursor.js:95
    smartUpdateChildren @ cursor.js:21
    visitJsxNode @ render.js:12
    render @ render.public.js:40
   (anonymous) @ index.html?html-proxy&index=0.js:6
Uncaught (in promise) TypeError: Failed to fetch dynamically imported module: http://localhost:3000/src/h_main_main_onmount.js
    Promise.then (async)
    then @ promises.js:27
    render @ render.public.js:40
    (anonymous) @ index.html?html-proxy&index=0.js:6
    client.ts:58 [vite] connected.
```

I tried to run the exact same on WSL2 (Ubuntu distro) on the Same Windows machine, and the demo ran without a problem.

**Comment by lovetingyuan** [comment]
+1

**Comment by lovetingyuan** [comment]
You must forget to use `normalizePath` offered by vite 😂.

**Comment by SebasG22** [comment]
+1 I can confirm that is not working on Windows and showing this error on the console dev tools.
![image](https://user-images.githubusercontent.com/17608169/163187117-38e63c49-44c7-481a-b283-b3a5a6b4d86e.png)

**Comment by AgentRBY** [comment]
+1

**Comment by JoshDollinger** [comment]
+1

**Comment by eglove** [comment]
Just to add to this, you currently need to use WSL to work with Qwik on Windows.

https://docs.microsoft.com/en-us/windows/wsl/install

TL;DR

wsl --install

As of recent updates, this will add Ubuntu as your default WSL distro.

**Comment by dzearing** [comment]
The issue is within the vite plugin; a number of calls to `path.resolve` and `path.join` should simply convert backslashes to forward. Ideally done in one place which can keep paths predictable. I was able to get to a working state locally. @adamdbradley is prepping a fix.

**Comment by mhevery** [comment]
I believe this has been resolved. Closing as fixed.
